mindmeld/query_factory.py

```diff
@@ -130,7 +137,8 @@ def create_query_factory(
         Returns:
             QueryFactory: A QueryFactory object that is used to create Query objects.
         """
-        del app_path
         tokenizer = tokenizer or Tokenizer.create_tokenizer()
         stemmer = stemmer or EnglishNLTKStemmer()
```

Comment:
Do we pick the stemmer based on the language & locale? Should we continue to use the English Stemmer for non-English languages? @serapio

Reply:
This does seem weird now. It should only default to the English stemmer if the language is English, and otherwise the default stemmer should be a dummy (no op) stemmer. We should probably also make the default tokenizer less opinionated (in future work). For other languages, the app should generally be defining it's own tokenizer and stemmer, but we shouldn't use the English stemmer if they fail to do so.

Reply:
@serapio What are your thoughts on using the Snowball stemmer?

```
In [1]: from nltk.stem.snowball import SnowballStemmer

In [2]: SnowballStemmer.languages
Out[2]: 
('arabic',
 'danish',
 'dutch',
 'english',
 'finnish',
 'french',
 'german',
 'hungarian',
 'italian',
 'norwegian',
 'porter',
 'portuguese',
 'romanian',
 'russian',
 'spanish',
 'swedish')
```

Reply:
Implemented the above suggestion